crypto/hd/hdpath.go

```diff
@@ -17,13 +17,15 @@ import (
 // To receive a canonical string representation ala
 // m / purpose' / coinType' / account' / change / addressIndex
 // call String() on a BIP44Params instance.
+/*
 type BIP44Params struct {
 	Purpose      uint32 `json:"purpose"`
 	CoinType     uint32 `json:"coinType"`
 	Account      uint32 `json:"account"`
 	Change       bool   `json:"change"`
 	AddressIndex uint32 `json:"addressIndex"`
 }
+*/
```

Comment:
Since I use [BIP44Params](https://github.com/cosmos/cosmos-sdk/blob/kerying-migration/crypto/hd/hd.pb.go#L26) from protos, I go ahead and remove this commented structure.